cot-cli/src/main.rs

```diff
@@ -33,8 +33,13 @@ enum Commands {
         #[arg(long)]
         name: Option<String>,
         /// Use the latest `cot` version from git instead of a published crate
+        /// (conflicts with --cot-path)
```

Comment:
`clap` has features to uphold that restraint, please use them

Reply:
Fair, I forgot they exist. Done.